pkg/core/blockchain.go

```diff
@@ -421,7 +421,10 @@ func (bc *Blockchain) storeBlock(block *Block) error {
 			vm.RegisterInteropFuncs(systemInterop.getSystemInteropMap())
 			vm.RegisterInteropFuncs(systemInterop.getNeoInteropMap())
 			vm.LoadScript(t.Script)
-			vm.Run()
+			_, err := vm.Run()
+			if err != nil {
+				log.WithError(err)
```

Comment:
Now you're logging it twice, here and 10 lines below.

Reply:
ok I see, add err to log statement